src/clj_http/conn_mgr.clj

```diff
@@ -152,16 +143,12 @@
                    (get-keystore-scheme-registry config)
 
                    :else regular-scheme-registry)]
-    (PoolingClientConnectionManager.
-     registry timeout java.util.concurrent.TimeUnit/SECONDS)))
-
-(def dmcpr ConnPerRouteBean/DEFAULT_MAX_CONNECTIONS_PER_ROUTE)
+    (PoolingHttpClientConnectionManager. registry)))
```

Comment:
Thanks, I made a pull request to fix it https://github.com/dakrone/clj-http/pull/322. Please note that even though the time to live is set, closeExpiredConnections must still be used to close expired connection. I guess this is because of this http://hc.apache.org/httpcomponents-client-4.3.x/tutorial/html/connmgmt.html#d5e405